client/my-sites/site-settings/form-seo.jsx

```diff
@@ -389,7 +392,21 @@ export const SeoForm = React.createClass( {
 					</Notice>
 				}
 
-				{ showUpgradeNudge && <SeoSettingsUpgradeNudge { ...{ upgradeToBusiness } } /> }
+				{ showUpgradeNudge &&
+					<SeoSettingsUpgradeNudge
+						plan={ PLAN_BUSINESS }
+						upgrade={ upgradeToBusiness }
+						title={ this.translate( 'Upgrade to a Business Plan and Enable Advanced SEO' ) }
+						subtitle={ this.translate( 'By upgrading to a Business Plan you\'ll enable advanced SEO features on your site.' ) }
```

Comment:
I think we prefer double quotes `"` if there's a single quote `'` chart in the text so we don't have to escape. :)